docs/docusaurus/docs/cloud/expectations/manage_expectations.md

```diff
@@ -49,8 +45,12 @@ The following table lists the available GX Cloud Expectations.
 | Uniqueness         | **column values to be unique**                       | Expect each column value to be unique.                                                                                                 | No                  |
 | Uniqueness         | **compound columns to be unique**                    | Expect the compound columns to be unique.                                                                                              | No                  |
 | Uniqueness         | **select column values to be unique within record**  | Expect the values for each record to be unique across the columns listed. Note that records can be duplicated.                         | No                  |
-| Validity           | **column value lengths to be between**                | Expect the column entries to be strings with length between a minimum value and a maximum value.                                       | No                  |
-| Validity           | **column value lengths to equal**                     | Expect the column entries to be strings with length equal to the provided value.                                                       | No                  |
+| Validity           | **column most common value to be in set**            | Expect the most common value to be within the designated value set.                                                                    | No                  |
+| Validity           | **column pair values to be equal**                   | Expect the values in column A to be the same as column B.                                                                              | No                  |
+| Validity           | **column value lengths to be between**               | Expect the column entries to be strings with length between a minimum value and a maximum value.                                       | No                  |
+| Validity           | **column value lengths to equal**                    | Expect the column entries to be strings with length equal to the provided value.                                                       | No                  |
+| Validity           | **column values to be in set**                       | Expect each column value to be in a given set.                                                                                         | No                  |
+| Validity           | **column values to not be in set**                   | Expect column entries to not be in the set.                                                                                            | No                  |
```

Comment:
I took some liberties with the alphabetical order to keep  **column values to be in set** and **column values to not be in set**  next to each other since these are 2 sides of the same coin